src/pages/iep/[iep_id].tsx

```diff
@@ -0,0 +1,76 @@
+import { trpc } from "@/client/lib/trpc";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import styles from "../../styles/Home.module.css";
+import Goals from "../../components/Goal";
+
+interface IepProps {
+  iep_id: string | string[] | undefined;
+}
+
+const Iep: React.FC<IepProps> = ({ iep_id }) => {
+  const router = useRouter();
+  const utils = trpc.useContext();
+
+  if (!iep_id) {
+    iep_id = router.query.iep_id;
+  }
```

Comment:
function parameters generally shouldn't be modified

```suggestion
  const {iep_id} = router.query
```

Reply:
Done